Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+
+let package = Package(
+    name: "JitsiMeetSDK",
+    platforms: [.iOS(.v14)],
+    products: [
+        .library(
+            name: "JitsiMeetSDK",
+            targets: ["JitsiMeetSDKWrapper"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/jitsi/webrtc", branch: "master"),
```

Comment:
Yes !

Reply:
That is not great. It needs to pick one that matches, like the Podfile does. Currently that is 118.x.x. How can we do that for SPM?

Reply:
Unfortunately this was my only option since the released version 118.0.0 doesn't have any Package.swift and since this version doesn't have any I can't specify it.

Reply:
Since it's us releasing the package, too, what would be the right thing to do here? Create a branch for each Major WebRTC release perhaps? I could do a spm-118, spm-119 etc.

Reply:
I don't know what the best strategy here, I think it's your decision.
What I just see it's that the Package.swift is not part of the release 118.0.0 (https://github.com/jitsi/webrtc/blob/v118.0.0/Package.swift), and it's weird because the file has been created before releasing the version.

Reply:
Yes, but it's created on the master branch, which doesn't have the code, but the artifact helpers. Maybe that is a bad choice.

The problem I see with making it part of each release is that unlike CocoaPods, you wouldn't be able to specify 118-the-latest, right?

How about putting Package.swift in the release branch? In the 118 case it will be M118. So, you can depend on the package in that branch, that will always give you the latest 118 version.

master can keep pointing at the latest-latest.

Does this make sense?

Reply:
No we have different option, that you can check here : 
- The from : https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#:~:text=.xcworkspace.-,Methods,-///%20Create%20a%20package

> /// The following example allows the Swift Package Manager to select a version
> /// like a  `1.2.3`, `1.2.4`, or `1.3.0`, but not `2.0.0`.
> ///
> ///    .package(url: "https://example.com/example-package.git", from: "1.2.3")

- And we also have methods like .upToNextMajor, .upToNextMinor that you can check here : https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#:~:text=of%20your%20package.-,Methods,-///%20Returns%20a%20requirement

Reply:
That sounds useful! How does it know where to look for the versions though? We only have https://github.com/jitsi/webrtc/blob/master/Package.swift Which now I realize doesn't even have the version number in it. 

Same thing applies here, actually. How would a consumer of this Package.swift know the right version to pick?

Reply:
It's linked to the release tag

Reply:
Oh, ok so it needs to be part of the branch and the release. Got it!

Reply:
I'll make a new release next week with it then.

Reply:
Perfect, thanks, will update this PR when it's done then 👍